core/arch/arm/kernel/secure_partition.c

```diff
@@ -53,9 +258,33 @@ static TEE_Result secure_partition_init(void)
 		else
 			msg[0] = '\0';
 		DMSG("SP %pUl size %u%s", (void *)&sp->uuid, sp->size, msg);
-	}
 
+		res = sp_init_uuid(&sp->uuid);
+
+		if (res != TEE_SUCCESS) {
+			EMSG("Failed initializing SP(%pUl) err: 0x%x"
+					, &sp->uuid, res);
+#ifndef SKIP_FAILED_SP
```

Comment:
This looks like debug code used while developing this. I don't think we need it any longer.

Reply:
I still want it as an option to be able to continue when one of the SPs fails